content/posts/android-tv-13.md

```diff
@@ -0,0 +1,92 @@
+---
+title: Android TV v0.13
+subtitle: That was quick
+author: Niels van Velzen
+githubusername: nielsvanvelzen
+date: 2022-03-14
+justify: center
+---
+
+<!-- markdownlint-disable MD033 MD036 -->
+
+We're back with another release for Android TV and Fire TV users containing a lot of bugfixes!
+
+<!--more-->
+
+As promised in the previous release post, updates are getting quicker. No more waiting for 2 years anymore. Today's update brings some new features and greatly improves playback of videos. Let's get started with the player changes.
+
+## Playback changes
+
+Most changes in this release are related to playing video files, the so called playback code. A lot of changes were made, both bugfixes and new features! The full list is, as always, available in the changelog. The most notable changes are listed below.
+
+## Transcoding updates
+
+A lot of issues with video are caused by the way the app transcodes video. We've tweaked the transcoding behavior and found a lot of issues were fixed with those tweaks, issues like the video ending early or unexpectedly rewinding itself. Unfortunately these changes triggered some issues in Jellyfin 10.7 and we had to limit these changes to servers running Jellyfin 10.8. We recommend updating as soon as a stable server release is ready.
+
+Additionally, we've added support for more audio codecs. The added codecs include Dolby Digital, Digital Plus and TrueHD, as well as DTS and Linear PCM audio, all commonly found on DVDs and Blu-rays, but also a more universal AAC support. This change is comparable with the audio codec changes in the last release of our mobile app.
+
+### Version selection
```

Comment:
this button is currently broken lol

Reply:
I tested this just now (latest master) and it works perfectly fine for me. I have multiple versions of big bunny. One of them is stereoscopic and switching the version just works.